x-pack/solutions/security/plugins/security_solution/public/flyout/ui_tests/tests/alert_details_url_sync.spec.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, test } from '@kbn/scout-security';
+import { CUSTOM_QUERY_RULE } from '@kbn/scout-security/src/playwright/constants/detection_rules';
+
+const RIGHT = 'right';
+
+test.describe('Expandable flyout state sync', { tag: ['@ess', '@svlSecurity '] }, () => {
+  test.beforeEach(async ({ browserAuth, detectionRuleApi }) => {
+    await detectionRuleApi.deleteAll();
+    await detectionRuleApi.createCustomQueryRule();
+    await browserAuth.loginAsPlatformEngineer();
+  });
+
+  test('should test flyout url sync', async ({
+    pageObjects: { alertsTablePage, alertDetailsRightPanelPage },
+  }) => {
+    await alertsTablePage.navigate();
+
+    const urlBeforeAlertDetails = await alertsTablePage.getCurrentUrl();
+    expect(urlBeforeAlertDetails).not.toContain(RIGHT);
```

Comment:
Just FYI: ESLint rule is checking for at least 1 `expect` assertion per test, so you are doing it correct not wrapping in page objects ❤️